docs/user-guide/aes-operator.md

```diff
@@ -0,0 +1,127 @@
+# The Ambassador Edge Stack Operator
+
+The Ambassador Edge Stack Operator is a Kubernetes Operator that controls the
+complete lifecycle of your Ambassador installation in your cluster. It also
+automates many of the repeatable tasks you have to perform for the Ambassador
+Edge Stack. Once installed, the AES Operator will automatically complete rapid
+installations, seamless upgrades to new versions of Ambassador, Azure
+integrations, and provides a reference architecture for ease of use. [Read more](https://github.com/datawire/ambassador-operator/blob/master/README.md#version-syntax).
```

Comment:
What do you mean by "Azure integrations" here? Also, the reference architecture isn't specific to the operator, it's specific to Azure...

Reply:
resolved